cmd/frakti/frakti.go

```diff
@@ -44,6 +58,8 @@ var (
 		"The directory for putting cni configuration file")
 	cniPluginDir = flag.String("cni-plugin-dir", "/opt/cni/bin",
 		"The directory for putting cni plugin binary file")
+	alternativeRuntimeEndpoint = flag.String("alternative-runtime", "unix:///var/run/docker.sock",
```

Comment:
Maybe it's better to just name it as  `docker-endpoint` since it only works for docker.